Server/Utils/word/AllowedRunes.go

```diff
@@ -0,0 +1,5 @@
+package wordUtils
+
+func VisibleRunes() []rune {
+	return []rune{'-', ' '}
```

Comment:
but those runes are not invisible 🤔

Reply:
I ment to say that those runes will be visible without guessing.
Do you have a better name?